test/e2e/core/setCellMeta.spec.js

```diff
@@ -109,12 +109,14 @@ describe('Core.setCellMeta', () => {
     expect(spec().$container.find('tbody tr:eq(1) td:eq(1)')[0].className).toEqual(classNames[0]);
   });
 
-  it('should call `afterSetCellMeta` plugin hook with visual indexes as parameters', () => {
+  it('should call `beforeSetCellMeta` and `afterSetCellMeta` plugin hook with visual indexes as parameters', () => {
```

Comment:
I'd add another test to check if the same coordinates are passed to both hooks when the index mapper internally converts them to different values.

Reply:
Yesterday we agreed that both hooks get the raw row and column. So I suppose that this proposes to additional check isn't necessary, is it?

Reply:
@pnowak True 👍